packages/rocketchat-message-star/client/views/starredMessages.coffee

```diff
@@ -1,13 +1,20 @@
 Template.starredMessages.helpers
 	hasMessages: ->
-		return StarredMessage.find({ rid: Session.get 'openedRoom' }, { sort: { ts: -1 } }).count() > 0
+		return StarredMessage.find({ rid: this.rid }, { sort: { ts: -1 } }).count() > 0
 
 	messages: ->
-		return StarredMessage.find { rid: Session.get 'openedRoom' }, { sort: { ts: -1 } }
+		return StarredMessage.find { rid: this.rid }, { sort: { ts: -1 } }
 
 	notReadySubscription: ->
 		return 'notready' unless Template.instance().subscriptionsReady()
 
 Template.starredMessages.onCreated ->
-	this.autorun =>
```

Comment:
That change wasn't supposed to be part of this PR. I will update it and resubmit.